src/com/codeenginestudio/bookManagement/servlets/EditBookServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.codeenginestudio.bookManagement.servlets;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.codeenginestudio.bookManagement.manage.ManageBook;
+import com.codeenginestudio.bookManagement.models.Book;
+import com.codeenginestudio.bookManagement.util.BookUtil;
+import com.codeenginestudio.bookManagement.validator.BookValidator;
+
+@WebServlet("/EditBookServlet")
+public class EditBookServlet extends HttpServlet {
+
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+
+		int bookId = Integer.parseInt(request.getParameter("bookId"));
```

Comment:
What happens when I put a `bookId` that does not exist! You can try it by changing the `bookId` on the browser url! I want to show an error page notice that "Book not found!".